pages/index.vue

```diff
@@ -93,6 +98,11 @@
 
         computed: {
 
+            ...mapGetters([
```

Comment:
@surchs This `mapGetters` can also be removed as the `getColumnNames` getter will be used by the `setDataTable` action in the store.